dgl_ptm/dgl_ptm/trade_money.py

```diff
@@ -1,7 +1,75 @@
-#!/usr/bin/env python
-# coding: utf-8
+import os
+import torch 
+import dgl 
+import dgl.function as fn
 
-# trade_money - Trades money between the different connected agents
+os.environ["DGLBACKEND"] = "pytorch"
```

Comment:
I understand that we need it here if we want to simply call this stand-alone for the time being, but this is sth. that would be set in the main function as it sets global environment variables at the OS level, if we leave it in here we could get conflicts somewhere down the line

Reply:
Yeah this makes sense. I have removed the line now. We can add it in main based on the user chosen backend.